projects/can_transceiver/src/can_transceiver.cpp

```diff
@@ -5,22 +5,31 @@
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 
+#include <chrono>
 #include <cstring>
+#include <functional>
+#include <memory>
 #include <stdexcept>
+#include <string>
 #include <thread>
 
 #include "can_frame_parser.h"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+using namespace std::chrono_literals;
 
 using IFreq       = struct ifreq;
 using SockAddr    = struct sockaddr;
 using SockAddrCan = struct sockaddr_can;
 
+#define QUEUE_SIZE 10
```

Comment:
Avoid using C-style define macros when possible. I would replace this with a `constexpr int QUEUE_SIZE = 10;`

Reply:
I realise this is no longer used in that .cpp file, so I've removed it instead